include/capstone.h

```diff
@@ -118,7 +120,8 @@ typedef struct cs_opt_mem {
 
 // Runtime option for the disassembled engine
 typedef enum cs_opt_type {
-	CS_OPT_SYNTAX = 1,	// Assembly output syntax
+	CS_OPT_NONE = 0,	// No opetion specified
```

Comment:
No problem. Probably I should have somehow minimized and/or summarized my
changes for ease of review. Feel free to request me anything, I am happy to
collaborate and make it easy to use CS from drivers.

On Thu, Apr 7, 2016 at 9:01 PM, Nguyen Anh Quynh <notifications@github.com>
wrote:

> In include/capstone.h
> <https://github.com/aquynh/capstone/pull/620#discussion_r58977884>:
>
> > @@ -118,7 +120,8 @@ typedef struct cs_opt_mem {
> >
> >  // Runtime option for the disassembled engine
> >  typedef enum cs_opt_type {
> > -	CS_OPT_SYNTAX = 1,	// Assembly output syntax
> > +	CS_OPT_NONE = 0,	// No opetion specified
>
> i am still looking at this PR. sorry for taking this too long, but too
> many things coming up now
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/aquynh/capstone/pull/620/files/6dad56669b1e2fb2f09484adfb5c494285204d18#r58977884>
>